### PR TITLE
Some minor docs improvements and examples for nf-test migration

### PR DESCRIPTION
**Pull request by jfy133**
After trying to run through it myself, and adding answers to provblems I found

### CODE REVIEW

File: src/content/docs/contributing/modules.md [hunk 1]
```diff
@@ -403,7 +415,8 @@ tag "<tool>/<sub-tool>" (optional)
 multiple tags are allowed for a test
 :::
 
-- Provide a test name preferably indicating the test-data and file-format used. Example: `test("homo_sapiens - [bam, bai, bed] - fasta - fai")`
+-  Provide a test name preferably indicating the test-data and file-format used. Example: `test("homo_sapiens - [bam, bai, bed] - fasta - fai")`
+  - For a single test, you can simply replace 'Should run without failures' boilerplate name.
```
Comment (mashehu) [inline src/content/docs/contributing/modules.md:419]:
```suggestion
  - For a single test, you can simply replace the 'Should run without failures' boilerplate name.  
```

**Review by mashehu — APPROVED** [review]

**Review by nvnieuwk — APPROVED** [review]
Thank you!

**Comment by jfy133** [comment]
@nf-core-bot fix linting

**Comment by mashehu** [comment]
the fix linting action, is broken at the moment 😬 need to run `pre-commit run` manually

**Comment by jfy133** [comment]
oh poop.

I tried, but the German mobile network currently says NEIN.

**Comment by jfy133** [comment]
Thanks @mashehu !

**Review by sateeshperi — APPROVED** [review]

**Pull request merged**